ui/src/submissions/jobs/containers/__tests__/StatusFieldContainer.test.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { fromJS } from 'immutable';
+
+import { getStoreWithState } from '../../../../fixtures/store';
+import StatusFieldContainer from '../StatusFieldContainer';
+
+describe('StatusField', () => {
```

Comment:
Since we're changing how we test containers, it's OK to remove it for now and add it later. (Maybe you can create a small task for this)